Reference/Searching/Examine/Quick-Start/index.md

```diff
@@ -9,13 +9,11 @@ _This guide will help you get set up quickly using Examine with minimal configur
 
 ## Performing a search
 
-In the coming examples, the Umbraco Starter Kit has been used, as it provides some example content that can be searched. Some of the examples below therefore may require 'the setting up of templates, etc' if you are following the guide on your existing site. You can use the following command to install the package: 
-
-```
-dotnet add package Umbraco.TheStarterKit
-```
+:::note
+In the coming examples the Umbraco Starter Kit has been used, as it provides some example content that can be searched. Some of the examples below therefore may require 'the setting up of templates, etc' if you are following the guide on your own existing site.
```

Comment:
Would be nice with a link to the starter kit docs here, since the installation instructions have (rightfully) been removed. 

```suggestion
In the coming examples the [Umbraco Starter Kit](https://our.umbraco.com/packages/starter-kits/the-starter-kit/) has been used, as it provides some example content that can be searched. Some of the examples below therefore may require 'the setting up of templates, etc' if you are following the guide on your own existing site.
```